Real-to-Virtual version 1.0 Documentation

bradleynull opened the pull request:
This implements a couple quick start guides for real-to-virtual. There's a data collection quick start which is intended to go through all the steps necessary to set up sensors, collect data, and parse it into the KITTI format. There's also a data processing quick start which roughly goes through the data processing pipeline. Since the processing pipeline is still a WIP, the documentation will need to be updated once the final products is complete.

Closes #90

citlaligm commented on docs/r2v/detailed_guide.md:
```diff
@@ -0,0 +1,164 @@
+
+1. Select the path where the logs are going to be stored using the “Directory 
+to Log” control: 
+    a. The program will create the following directory structure to store your data:
```
Missing a new line 
![image](https://user-images.githubusercontent.com/10260895/79772863-41756600-82f6-11ea-8884-0e3ae48b3ba0.png)

bradleynull replied:
![Screenshot from 2020-04-20 12-03-27](https://user-images.githubusercontent.com/6026642/79778831-fd3a9380-82fe-11ea-9479-fdac465d2241.png)
Hmmm this is rendering correctly for me... let me know after you pull if it still doesn't look right to you.

citlaligm commented on docs/r2v/detailed_guide.md:
```diff
@@ -0,0 +1,164 @@
+
+1. When you are ready to finish collecting data, click the EXIT button, this will disconnect the GNSS and close the open connections.
+
+    **NOTE:** If the the LabView “Stop” button is pressed instead of the R2V EXIT button, the entire application will need to be restarted in order to record again.
```
extra _"the"_

bradleynull replied:
Good catch! Fixed.

bradleynull commented on docs/r2v/data_collection_quick_start.md:
```diff
@@ -0,0 +1,156 @@
+    <img class='lg_img' src="../imgs/kitti_data_full_size_image.png"/>
+</div>
+
+The data collection software provides a tool to determine the optimal 
```
Yea, I  figured that was the case, but I wanted to get this here to show that something existed or will exist. I'd vote to leave it in for now.

bradleynull commented on docs/r2v/data_collection_quick_start.md:
```diff
@@ -0,0 +1,156 @@
+transforms in a package that can be easily mounted to a roof mount:
+
+<div class="img_container">
+    <img class='lg_img' src="../imgs/real_to_virtual_hardware_side_view.png"/>
```
This image is in the repo and appears to work for me: https://github.com/monoDriveIO/documentation/blob/r2v_quickstart/docs/r2v/imgs/real_to_virtual_hardware_side_view.png

bradleynull commented on docs/r2v/data_collection_quick_start.md:
```diff
@@ -0,0 +1,156 @@
+to calibrate. 
+
+1. Open that Xsens Magnetic Field Mapper software
+1. Select "Write results to Motion Tracker non-volatile memory"
```
Thanks for looking that up. I was just going by what the Xsens documentation was saying.

Fixed to match.

bradleynull commented on docs/r2v/data_collection_quick_start.md:
```diff
@@ -0,0 +1,156 @@
+After calibration, **do not power off the hardware**. This will preserve all
+calibration parameters. To collect data:
+
+1. Click the "Run" arrow on the top-left of the monoDrive LabView data logger application. The data logger will connect to all sensors and ensure they are functioning as expected.
```
Fixed for all r2v documentation.

bradleynull commented:
@citlaligm Thanks for looking through this! I addressed all your comments and its ready for another look.

I've also added an additional page about using the UE level when complete. Please give that a look too. Thanks!

citlaligm commented on docs/r2v/data_collection_quick_start.md:
```diff
@@ -0,0 +1,156 @@
+</div>
+
+The data collection software provides a tool to determine the optimal 
+field-of-view in both the horizontal and vertial dimensions. 
```
Just see this other typo.
